build/webpack.js

```diff
@@ -90,4 +90,16 @@ module.exports = {
       loaders: [babel]
     }
   },
+  nodeBrowser: {
+    context: path('../src'),
+    entry: './runtime/node/browser.js',
```

Comment:
I'm not a fan of the naming here.  This isn't `node`, but rather `npm` if I understand correctly?  Also, in other target points we use `web` instead of `browser`.  How about we create `src/runtime/web/common.js` instead of `src/runtime/node/browser.js` and the dist file is called `dist/webcompat/l20n-common.js`?